application/src/main/java/org/togetherjava/tjbot/features/mediaonly/MediaOnlyChannelListener.java

```diff
@@ -45,7 +53,22 @@ public void onMessageReceived(MessageReceivedEvent event) {
         }
 
         if (messageHasNoMediaAttached(message)) {
-            message.delete().flatMap(any -> dmUser(message)).queue();
+            message.delete().flatMap(any -> dmUser(message)).queue(res -> {
+                logger.info("This is a Error handler of DM");
+            }, new ErrorHandler().handle(ErrorResponse.CANNOT_SEND_TO_USER, err -> {
+                logger.debug(err.toString());
+                logger.error(message.toString());
```

Comment:
what is this for? remove it

Reply:
@Zabuzard  had a suggestion to have a debug so shall remove just the Error?

Reply:
from what i understand he said either just log it or handle it, do one or the other and ur already handling it(sending message in channel) so remove both, also the logs message makes no sense

Reply:
ok Its Done i will add it on to the next push